api/cpp/stm32/slint-stm.h

```diff
@@ -0,0 +1,45 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+#pragma once
+
+#define STRINGIFY(X) STRINGIFY2(X)
+#define STRINGIFY2(X) #X
+#define CAT(X, Y) CAT2(X, Y)
+#define CAT2(X, Y) X##Y
```

Comment:
maybe rename this to add a prefix such as `SLINT_STM32_STRIGNIFY`

Reply:
Good idea